hadoop-project/pom.xml

```diff
@@ -1196,6 +1196,16 @@
         <artifactId>jsch</artifactId>
         <version>0.1.54</version>
       </dependency>
+      <dependency>
+        <groupId>io.opentracing</groupId>
+        <artifactId>opentracing-util</artifactId>
+        <version>0.31.0</version>
+      </dependency>
+      <dependency>
+        <groupId>org.apache.hadoop.thirdparty</groupId>
+        <artifactId>hadoop-shaded-jaeger</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
```

Comment:
use ${hadoop-thirdparty-protobuf.version}

Reply:
Done in 22cf60451b2910f6f00d9274e10a68c7042d3e8d